### ISSUE TITLE
Question: Socialist module is not visible on homepage

### ISSUE THREAD

**Issue by DanaReilly** [iatek/jquery-socialist#35]
Hi,

I am building a one-page website, divided into three horizontal rows.  I want to display the Socialist plugin within one of the rows, but the content is not showing up.  The plugin seems to only work when I specify the dimensions of the element '#content', but when I give this element a certain height or width, the isotope layout doesn't work.  What am I doing wrong?

Here is an example page:  danareilly.com.

Thanks for your help.

**Comment by kmusch** [comment]
I didnt dig to much but this is different then what I use. 

YOURS=
         jQuery(document).ready(function ($) {
     $('#content').socialist({ 
                networks: [ 

MINE=
    $(document).ready(function () {
        $('#content').socialist({
        networks: [

Also, I don't see the "fields" (ex: fields:['image','followers','likes'],

**Comment by DanaReilly** [comment]
Thank you for your help!  I figured out the problem (if fields aren't specified in the HTML head, the script just uses the default settings for fields... so that wasn't the problem.  It had something to do with the container I tried to use around the #content div).

Another question:  How can I filter the content that is pulled, so that every visible socialist element has an image (filter the ones without an image out)?  

+Is there a way to change the shape of the socialist boxes?  Similar to the way Isotope has customizable shapes for each element (depending on the type of content in the element)?  Thanks for your help!  

Same website with the same issues:  danareilly.com

: )
